### PR TITLE
Add features #626 and #627 and Fix #628

### LINKED ISSUE #628: Output file validation
**Is your feature request related to a problem? Please describe.**
I'm running into a problem where book being liberated are encountering errors, and it results in writing out partial m4b files. I'm still tracking down the cause of that, but it would be nice if there was the ability to validate the file was written correctly and not mark the book as liberated if it wasn't.

**Describe the solution you'd like**
I'd like the the ability to have Libation validate that the file was written correctly. This is more of a vague enhancement request. There are a couple points in the process that feel like they could be validated, when the book is initially downloaded and when the book is moved to the final destination. For the first one, it would largely depend on what kind of metadata you can get from audible. For the second one it could check that the file has the same checksum as the one in the temp folder.

**Describe alternatives you've considered**
I've considered writing a script to run periodically and flag books that are suspiciously small files.

### PR DESCRIPTION
**Pull request by Mbucari**
## [Add feature](https://github.com/rmcrackan/Libation/commit/46f6ba17101c70119c14284d806a4d3a60b7c162) #627 [and fix bug](https://github.com/rmcrackan/Libation/commit/46f6ba17101c70119c14284d806a4d3a60b7c162) #628

- Add setting to overwrite files when moving from temp folder to Books folder (#627)
- Do not set `LiberatedStatus` to `LiberatedStatus.Liberated` if moving files threw an exception (#628)

## [Write error info to StdErr](https://github.com/rmcrackan/Libation/commit/ec9854212a509013de338779cdf53b9739808c0d) #626

### CODE REVIEW

File: Source/LibationCli/Options/_OptionsBase.cs [hunk 1]
```diff
@@ -18,11 +18,11 @@ public async Task Run()
 			{
 				Environment.ExitCode = (int)ExitCode.RunTimeError;
 
-				Console.WriteLine("ERROR");
-				Console.WriteLine("=====");
-				Console.WriteLine(ex.Message);
-				Console.WriteLine();
-				Console.WriteLine(ex.StackTrace);
+				Console.Error.WriteLine("ERROR");
```
Comment (rmcrackan) [inline Source/LibationCli/Options/_OptionsBase.cs:21]: Good call. I always forget `Console.Error` exists

**Comment by Mbucari** [comment]
@rmcrackan Please see latest commit: [Fix Stupid](https://github.com/rmcrackan/Libation/pull/631/commits/7a9c6720c73f8a1ff3209342e90e03a4fd9df154)

**Comment by rmcrackan** [comment]
Since you've got everything open, do you want to [throw this in there too](https://github.com/rmcrackan/Libation/issues/629#issuecomment-1590434045)? I feel like it should be 1 line of code plus another line of markdown.

I was thinking this morning: a completely masturbatory backend task could be a post-build runner which auto-updates stuff like nuspec and the tags readme.

**Comment by rmcrackan** [comment]
> Fix Stupid

There's a saying around here "you can't fix 'stupid'" :P

**Comment by Mbucari** [comment]
> Since you've got everything open, do you want to [throw this in there too](https://github.com/rmcrackan/Libation/issues/629#issuecomment-1590434045)?

Sure thing.

> I was thinking this morning: a completely masturbatory backend task could be a post-build runner which auto-updates stuff like nuspec and the tags readme.

I'll keep that in my back pocket for when boredom strikes again.

> you can't fix 'stupid'

I'm an optimist :D

**Comment by rmcrackan** [comment]
> I'm an optimist :D

At every poker table, there is a fool. And if you can't find him, I've got some bad news...

**Comment by Mbucari** [comment]
#629 is done

**Pull request merged**